gtsfm/averaging/rotation/shonan.py

```diff
@@ -157,6 +168,7 @@ def run_rotation_averaging(
         num_images: int,
         i2Ri1_dict: Dict[Tuple[int, int], Optional[Rot3]],
         i1Ti2_priors: Dict[Tuple[int, int], PosePrior],
+        v_corr_idxs: Dict[Tuple[int, int], np.ndarray],
```

Comment:
We could just pass in the number of correspondences if we want to be efficient

Reply:
Acknowledged. I slightly prefer to stick with our core data types / primitives as the interface for APIs, to prevent extra processing in the main modules.